src/parser/jobs/drg/modules/Buffs.tsx

```diff
@@ -77,7 +77,9 @@ export default class Buffs extends Analyser {
 	override initialise() {
 		const playerFilter = filter<Event>().source(this.parser.actor.id)
 		this.addEventHook(playerFilter.type('action'), this.onCast)
-		this.addEventHook(playerFilter.type('statusApply').status(this.data.statuses.RIGHT_EYE_SOLO.id), () => this.soloDragonSight = true)
+
+		this.addEventHook(filter<Event>().source(this.parser.actor.id).action(this.data.actions.DRAGON_SIGHT.id), this.onDragonSight)
```

Comment:
Minor nit, but is there any particular reason not to reuse `playerFilter` here? It looks to be the same base filter as the other hooks.